templates/aws-stack.yml

```diff
@@ -268,6 +269,11 @@ Parameters:
     Description: Optional - Security group id to assign to instances
```

Comment:
should we put "DEPRECATED" here? Or is it possible that we might not deprecate it?